src/Geekhub/DreamBundle/Model/DreamsResponse.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+namespace Geekhub\DreamBundle\Model;
+
+use JMS\Serializer\Annotation\ExclusionPolicy;
+use JMS\Serializer\Annotation\Expose;
+use JMS\Serializer\Annotation\Type;
+
+/**
+ * Class DreamResponse
+ * @package Geekhub\DreamBundle\Model
+ * @ExclusionPolicy("all")
+ */
+class DreamsResponse
```

Comment:
move all properties (without $deams) in Abstracion class and its getters use to implement PaginationInterface